src/dcap_provider.cpp

```diff
@@ -39,6 +40,13 @@ constexpr char TCB_INFO[] = "SGX-TCBm";
 constexpr char CONTENT_TYPE[] = "Content-Type";
 constexpr char QE_ISSUER_CHAIN[] = "SGX-QE-Identity-Issuer-Chain";
 constexpr char REQUEST_ID[] = "Request-ID";
+constexpr char CACHE_CONTROL[] = "Cache-Control";
+
+    namespace cache_control
+    {
+        // as of the initial API version, max-age is the only supported header

```

Comment:
This is a little confusing since it doesn't look like the DCAP provider even parses a Cache-Control policy from the header currently. What does the max-age policy mean and does it matter in the absence of other policies?

Reply:
Hmm, I think Seth had it in here as a stub for eventually having the max age of the local cache be policy configurable. Currently it just sets everything to expire in a day on the local cache, see **get_cache_max_age**